This prevents fatal error when targeting 32-bit:
glslopt@0.1.11: clang: error: unsupported option '-mfpu=' for target 'x86_64-unknown-linux-gnu'
('-mfpu' argument, and other invalid argumnets, come from termux_setup_toolchain_28c,
but are important to preserve for C++ code, while hiding them from Rust code)

--- a/config/makefiles/rust.mk
+++ b/config/makefiles/rust.mk
@@ -10,6 +10,9 @@
 # commands can be executed directly by make, without doing a round-trip
 # through a shell.
 
+unexport CFLAGS
+unexport CXXFLAGS
+
 cargo_host_flag := --target=$(RUST_HOST_TARGET)
 cargo_target_flag := --target=$(RUST_TARGET)
 
